build.gradle

```diff
@@ -55,7 +55,7 @@ ext {
     zstdVersion = '1.5.5-3'
     awsSdkV2Version = '2.18.9'
     awsSdkV1Version = '1.12.337'
-    mongoCryptVersion = '1.8.0'
+    mongoCryptVersion = '1.8.0-SNAPSHOT'
```

Comment:
Can this be `1.10.0-SNAPSHOT` now?

Reply:
I did not bump the version here, it is using the already-published-but-misnamed version.

I bumped the version in mongo crypt, but did not publish. I could do this, but then we will have to publish the next version and update this regardless, before the next release. I added https://jira.mongodb.org/browse/JAVA-5492 to track updating the version.

Reply:
It looks like a 1.10.0-SNAPSHOT has already been published by  [Evergreen](https://spruce.mongodb.com/task/libmongocrypt_java_release_publish_java_9347305b735d643a043abaa7775ff16779f8cdb1_24_06_04_19_51_47/logs?execution=0), so I expect it will work if you bump it.

Reply:
Ah, I see.